scheme/sevsnp/evidence_handler_test.go

```diff
@@ -0,0 +1,127 @@
+// Copyright 2025 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package sevsnp
+
+import (
+	"encoding/json"
+	"github.com/stretchr/testify/assert"
+	"github.com/veraison/ear"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/veraison/services/proto"
+)
+
+var testNonce = []byte{
+	0x07, 0x06, 0x05, 0x04, 0x03, 0x02, 0x01, 0x00,
+	0x0f, 0x0e, 0x0d, 0x0c, 0x0b, 0x0a, 0x09, 0x08,
+	0x17, 0x16, 0x15, 0x14, 0x13, 0x12, 0x11, 0x10,
+	0x1f, 0x1e, 0x1d, 0x1c, 0x1b, 0x1a, 0x19, 0x18,
+}
+
+func Test_ExtractClaims_ok(t *testing.T) {
```

Comment:
Just checking, I did not see the Full Evidence JSON Claims set in the test cases..?

is there any reason for this..?